src/RazorSdk/SourceGenerators/RazorSourceGeneratorEventSource.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.NET.Sdk.Razor.SourceGenerators
+{
+    [EventSource(Name = "Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator")]
```

Comment:
The standard pattern is to use dashes in the EventSource name (e.g. Microsoft-DotNet-SDK-Razor-SourceGenerator or something like that).